build/make-initrd.sh

```diff
@@ -9,6 +9,18 @@ cp iptables /tmp/hyperstart-rootfs
 cp libm.so.6 /tmp/hyperstart-rootfs/lib64/
 tar -xf modules.tar -C /tmp/hyperstart-rootfs/lib/modules
 
+# create symlinks to busybox and iptables
+for bin in ${BUSYBOX_BINARIES}
+do
+	mkdir -p /tmp/hyperstart-rootfs/`dirname ${bin}`
+	ln -s /busybox /tmp/hyperstart-rootfs/${bin}
+done
```

Comment:
could be, but what about adding an option to enable/disable host path? if enable then use host paths otherwise use predefined paths (/tar, /sh, ....)
I'd like to use hyperstart as daemon into a custom image but I would not like to have busybox and symlinks to it